hugo.toml

```diff
@@ -1,14 +1,14 @@
-baseURL = "/"
+baseURL = "https://about.ipfs.io/"
 themesDir = "themes"
```

Comment:
Is this actually necessary, was anything not working because of this? 
I see you're linking to image assets below using this, that will mean that they come from the published site instead of the build preview or local assets. And if we ever have sub-pages, this will likely break the preview and dev sites generated by Vercel.